packages/calypso-color-schemes/src/calypso-color-schemes.scss

```diff
@@ -12,3 +12,4 @@
 @import 'shared/color-schemes/powder-snow';
 @import 'shared/color-schemes/sakura';
 @import 'shared/color-schemes/sunset';
+@import 'shared/color-schemes/jetpack-cloud';
```

Comment:
I think the eventual plan would be to have a jetpack-cloud colour scheme. cc @folletto. I didn't want to create one just yet. Since that should probably be its own PR.

Reply:
Yes, what Enej said: we need to have the Jetpack color scheme available to anyone in Calypso/wp (the WordPress·com picker above), which should be the one loaded also on Calypso/jp (Jetpack Cloud) by default.
And yes, should be a separate PR.

Reply:
Gotcha, TIL 👍 In that case, I'm happy to leave it as-is.